README.md

```diff
@@ -190,6 +190,9 @@ require'nvim-tree'.setup { -- BEGIN_DEFAULT_OPTS
       global = false,
       restrict_above_cwd = false,
     },
+    expand_all = {
+      max_folder_discovery = 300,
```

Comment:
not too sure about the option name here.

Reply:
`max_folder_expand` ?

Reply:
we'll see if this makes sense for user as it is.